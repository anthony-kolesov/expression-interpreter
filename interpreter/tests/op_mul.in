# Typical operation.
out 5 * 3
print "\n"

# Relatively big numbers.
out 1000000 * 2
print "\n"

# Crossing the zero.
out -4 * -9
print "\n"

# Staying negative.
out -9 * 5 
print "\n"

# Floating point numbers.
out 3.4 * 1.2
print "\n"

# Floating point numbers crossing zero.
out -3.4 * 1.2
print "\n"

# Zero values.
out 3 * 0
print "\n"

out 0 * 6
print "\n"

out 0 * 0
print "\n"

# Multiplication by one.
out 1 * 7
print "\n"

out 7 * -1
print "\n"

