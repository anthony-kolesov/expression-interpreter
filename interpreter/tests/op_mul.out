15
2000000
36
-45
4.080000
-4.080000
0
0
0
7
-7
